### DISCUSSION TITLE
Auto-close switch network modal

### DISCUSSION THREAD

**Discussion by brunowowk** [family/connectkit#105] [Ideas]
When the user is prompted to switch networks and does it, the modal stays open. This UX is a bit confusing, as the user could think that the X icon will disconnect him.

![image](https://user-images.githubusercontent.com/847070/216298563-b80acec4-6e23-4326-9d29-c3bf1a1a3034.png)

I think it would be best to have that modal auto-close upon switching to the right network, maybe with a few seconds of delay

**Comment by harry-nutsfi** [comment]
Wondering the same. This UX is confusing.

**Comment by ElRodrigote** [comment]
Bumping this thread again, anyone got a way around this modal not closing after a successful connection?
